.github/CODEOWNERS

```diff
@@ -933,6 +930,12 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+# ServiceLabel: %Network - CDN %Mgmt
+# ServiceOwners:                                                   @cdnfdsuppgithub
```

Comment:
The account `cdnfdsuppgithub` does not have the correct account memberships/permissions to be a CODEOWNERS contact.